Sources/Paywalls/Components/PaywallTextComopnent.swift

```diff
@@ -25,6 +26,7 @@ public extension PaywallComponent {
 
         public init(
             text: DisplayString,
```

Comment:
Note: We will need to get rid of this _eventually_ when all of our test paywalls are using the localized strings in their new place/format